backend/properties/README.md

```diff
@@ -1,30 +0,0 @@
-# Configuration properties
```

Comment:
I think this section could come handy on the backend readme.md

Reply:
Okay, I brought back the readme and improved it https://github.com/tsmean/tsmean/tree/fix/cors/backend/properties

Reply:
cool thanks